src/frontend/src/pages/index.js

```diff
@@ -40,12 +43,62 @@ export default function IndexPage() {
     getPosts(numPages);
   }, [telescopeUrl]);
 
+  useEffect(() => {
+    async function getPostsCount() {
+      try {
+        const res = await fetch('http://localhost:3000/posts');
+        // const res = await fetch(`${telescopeUrl}/posts`);
+        if (!res.ok) {
+          throw new Error(res.statusText);
+        }
+        return res.headers.get('x-total-count');
+      } catch (error) {
+        console.log(error);
+      }
+      return null;
+    }
+
+    // Get the current posts count when page loads
+    async function setPostsInfo() {
+      try {
+        await Promise.all([
+          await setInitNumPosts(await getPostsCount()),
+          await setCurrentNumPosts(await getPostsCount()),
+        ]);
+        console.log(initNumPosts);
+        console.log(currentNumPosts);
+      } catch (error) {
+        console.log({ error });
+      }
+    }
+
+    setPostsInfo();
+    // Polls every minute, this timer starts as soon as the page is loaded
+    const interval = setInterval(async () => {
+      await setCurrentNumPosts(await getPostsCount());
+      console.log(currentNumPosts);
+      if (currentNumPosts !== initNumPosts) {
+        console.log('Not the same!');
+      } else {
+        console.log('Same!');
+      }
+    }, 5000);
```

Comment:
This is probably happening too often.  If I'm on mobile, I don't want to send this much traffic.  I'd probably do once per minute, or more.

Reply:
I was thinking every 5 mins or so, but for testing purposes I set it as 5s